graphs/possible_bipartition.py

```diff
@@ -1,12 +1,36 @@
-# Can be used for BFS
-from collections import deque 
-
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(E)
```

Comment:
Time complexity will be O(N+E) where N is the number nodes and E is the number of edges since you should traverse each node and edge once.

Space complexity will be O(N) since that is the maximum number of recursive calls you could have in the stack at one time.